airflow/plugins/operators/sentry_to_gcs.py

```diff
@@ -179,22 +177,22 @@ def __init__(
         self.bucket = bucket
         self.project_slug = project_slug
         self.auth_token = auth_token
-        self.target_date = target_date
 
         super().__init__(**kwargs)
 
-    def execute(self, **kwargs):
-        ts = pendulum.now()
-        dt = ts.date()
+    def execute(self, context, **kwargs):
+        execution_ts = pendulum.from_format(context.get("ts"), "YYYY-MM-DDTHH:mm:ssZ")
+        execution_dt = execution_ts.date()
         extract = SentryExtract(
             project_slug=self.project_slug,
-            dt=dt,
-            ts=ts,
-            filename=f"{self.project_slug}.jsonl.gz",
+            execution_dt=execution_dt,
+            execution_ts=execution_ts,
+            filename=f"{execution_ts}.jsonl.gz",
```

Comment:
Have you thought about backfills and whether we want to always overwrite?

Also it looks like you end up setting `execution_ts` to a different value later; could you just delay instantiating the extract object until after the API scrape? I think the function signature could look something like:

```python
def fetch_and_clean_from_sentry(target_date: pendulum.Date, auth_token: str) -> Tuple[pendulum.DateTime, Optional[List[Dict]]]:
```

And then you create the extract object just before saving data.